### ISSUE TITLE
Boost headers missing [Mac OS X, terminal build]

### ISSUE THREAD

**Issue by wendles** [ja2-stracciatella/ja2-stracciatella#333]
$ make build-on-mac
[...]
```
===> CXX _build/lib-boost/libs/filesystem/src/path.cpp
_build/lib-boost/libs/filesystem/src/path.cpp:41:11: fatal error: 
      'boost/filesystem/detail/utf8_codecvt_facet.hpp' file not found
# include <boost/filesystem/detail/utf8_codecvt_facet.hpp>
          ^
1 error generated.
```
It appears that boost/filesystem/detail/* doesn't exist in tree; it may be necessary to include the entire directory of ../../detail/.

Please note too this is just where the build stopped, I have not yet determined if more headers are needed.

**Comment by selaux** [comment]
Please try again with the branch from #334.

**Comment by selaux** [comment]
@wendles Can you try again with #334?

**Comment by wendles** [comment]
Verified fixed, Mac OS X 10.11.5, Xcode tools 7.3.1.
